### DISCUSSION TITLE
routes/web.php works locally but not on server

### DISCUSSION THREAD

**Discussion by eliasu** [statamic/cms#5978] [Troubleshooting] [answered]
Hi there, 

I'm trying to implement a autogenerated sitemap on my page. Everything works fine locally, but as soon as I upload everything, it won't show the sitemap. I found that the routes within `routes/web.php` wouldn't work after the upload any more. I added this to demonstrate:
```php
Route::get('foo', function () { return 'foo'; });
```
… and even this is not working. 

Any Ideas on that? Could it be something with the PHP Version? 
Thanks!

**Comment by duncanmcclean** [comment, marked as answer]
Have you tried clearing your route cache?

```
php artisan route:clear
```

PHP version shouldn't make any difference.

**Reply by eliasu** [reply]
I didnt know, routes had a cache too 😔 I looked for hours, and this fixed it all. 
Thanks!

**Reply by duncanmcclean** [reply]
No worries - routes aren't cached by default but you probably did it if you ran `php artisan optimise` or something.
